etc/ci/performance/test_differ.py

```diff
@@ -49,4 +53,13 @@ def load_data(filename):
         diff = value2 - value1
         change = diff / value1
         color = BLUE if value1 <= value2 else GREEN
+        total1 += value1
+        total2 += value2
         print("{}{:6} {:6} {:+6} {:+8.2%}   {}.{}".format(color, value1, value2, diff, change, key, END))
+
+
+print
```

Comment:
Testing locally, this bare print appears to insert a blank line in python2 but it's a no-op in python3. You need `print()` to get the newline.

Reply:
(Bashes head on desk). Fixed.